.gitignore

```
@@ -1,3 +1,2 @@
 .idea folder
```

```suggestion
.idea/
```
Check: https://www.pluralsight.com/guides/how-to-use-gitignore-file